types/p-wait-for/v1/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for p-wait-for 1.0
+// Project: https://github.com/sindresorhus/p-wait-for#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = pWaitFor;
+
+declare function pWaitFor(condition: () => PromiseLike<boolean> | boolean, interval?: number): Promise<void>;
```

Comment:
```suggestion
declare function pWaitFor(condition: () => PromiseLike<boolean> | boolean, options?: { interval?: number, timeout?: number }): Promise<void>;
```
From the source, it looks like the options should be an object: https://github.com/sindresorhus/p-wait-for/blob/master/index.js

Reply:
I'm trying to restore typings for v1. What you read are v2 docs. I  linked the wrong docs in the description, [here](https://github.com/sindresorhus/p-wait-for/tree/v1.0.0) are the correct ones.